specification.md

```diff
@@ -3144,14 +3144,14 @@ reduce the number of connections to the Server when a very large number
 
 ### Configuration Management
 
-* [Uber Flipr](https://eng.uber.com/flipr/)
+* [Uber Flipr](https://web.archive.org/web/20240223111552/https://www.uber.com/blog/flipr/)
```

Comment:
The bare link is now https://www.uber.com/blog/flipr/, but hitting the link with cURL results in `406 Not Acceptable` so I opted for the web.archive.org link.

Reply:
Given that https://www.uber.com/blog/flipr exists, I'd vote to have an ignore pattern for this one.

Reply:
Ok, done in https://github.com/open-telemetry/opamp-spec/pull/196/commits/515ba8a89e7596121e2de554e9532550f9baac30